sql/2021/javascript/module_and_nomodule.sql

```diff
@@ -0,0 +1,25 @@
+SELECT
+  client,
+  COUNT(DISTINCT IF(module, page, NULL)) AS module,
+  COUNT(DISTINCT IF(nomodule, page, NULL)) AS nomodule,
+  COUNT(DISTINCT IF(nomodule AND module, page, NULL)) AS both,
+  COUNT(DISTINCT page) AS total,
+  COUNT(DISTINCT IF(module, page, NULL)) / COUNT(DISTINCT page) AS pct_module,
+  COUNT(DISTINCT IF(nomodule, page, NULL)) / COUNT(DISTINCT page) AS pct_nomodule,
```

Comment:
Could we drop this query?

We get % module/nomodule in `breakdown_of_scripts_using_async_defer_module_nomodule.sql`. Not sure if the response body / custom metric distinction is as meaningful for these metrics as async and defer.

Reply:
@rviscomi `module_and_nomodule.sql` records the % of pages using module/module which is what I am talking about in the chapter. `breakdown_of_scripts_using_async_defer_module_nomodule.sql` however, is percent of scripts. 

looking at percent of pages using response body 
module -4.6%
nomodule - 3.9%

percent of pages using custom metric
module -3.9%
nomodule - 0.004%

The custom metric results don't look right so I'd prefer sticking to `module_and_nomodule.sql` and talk about results from initial HTML

Reply:
> percent of pages using custom metric
>module -3.9%
>nomodule - 0.004%

Do you have the query for this? Or should [no]module data be added to `breakdown_of_pages_using_async_defer.sql`?

Also it looks like this query is looking at 2020 data. Should be updated to 2021.

Reply:
Updated to 2021 data.
added [no]module to `breakdown_of_pages_using_async_defer.sql` in a58141a